src/Test.rsc

```diff
@@ -17,3 +18,39 @@ void testMain(loc input) {
   TEnv tenv = collect(ast);
   set[Message] msgs = check(ast, tenv, g.useDef);
 }
+
+Value getValue(value val) {
+  switch (val) {
+    case int n:
+      return vint(n);
+    case bool b:
+      return vbool(b);
+    case str s:
+      return vstr(s);
+    default:
+      throw "Unsupported value <val>";
+  }
+}
+
+void testEval() {
```

Comment:
I would move this functionality into testMain.

Now it's just duplicating a bunch of work.